annotated-projects/PXD001774/PXD001774.sdrf.tsv

```diff
@@ -0,0 +1,19 @@
+source name	characteristics[organism]	characteristics[organism part]	characteristics[organism part]	characteristics[cell type]	characteristics[disease]	characteristics[biological replicate]	assay name	comment[proteomexchange accession number]	comment[data file]	comment[file uri]	comment[fraction identifier]	comment[technical replicate]	comment[label]	comment[instrument]	comment[cleavage agent details]	comment[modification parameters]	comment[modification parameters]	comment[modification parameters]	comment[modification parameters]	comment[modification parameters]	comment[precursor mass tolerance]	comment[fragment mass tolerance]
+Sample 1	Oryza sativa	Embryo	Plant cell	not applicable	not applicable	1	run 1	PXD001774	0HNP1timecourse.raw	http://ftp.pride.ebi.ac.uk/pride/data/archive/2016/03/PXD001774/0HNP1timecourse.raw	1	1	NT=label free sample	AC=MS:1000449;NT=LTQ Orbitrap	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	10 ppm	0.8 Da
+Sample 1	Oryza sativa	Embryo	Plant cell	not applicable	not applicable	2	run 2	PXD001774	0HNP2timecourse.raw	http://ftp.pride.ebi.ac.uk/pride/data/archive/2016/03/PXD001774/0HNP2timecourse.raw	1	1	NT=label free sample	AC=MS:1000449;NT=LTQ Orbitrap	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	10 ppm	0.8 Da
+Sample 1	Oryza sativa	Embryo	Plant cell	not applicable	not applicable	3	run 3	PXD001774	0HNP3timecourse.raw	http://ftp.pride.ebi.ac.uk/pride/data/archive/2016/03/PXD001774/0HNP3timecourse.raw	1	1	NT=label free sample	AC=MS:1000449;NT=LTQ Orbitrap	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	10 ppm	0.8 Da
+Sample 2	Oryza sativa	Embryo	Plant cell	not applicable	not applicable	1	run 4	PXD001774	12HNP1timecourse.raw	http://ftp.pride.ebi.ac.uk/pride/data/archive/2016/03/PXD001774/12HNP1timecourse.raw	1	1	NT=label free sample	AC=MS:1000449;NT=LTQ Orbitrap	AC=MS:1001251;NT=Trypsin	NT=Carbamidomethyl;AC=UNIMOD:4;TA=C;MT=Fixed	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Phospho;AC=UNIMOD:21;TA=S;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=T;MT=Variable	NT=Phospho;AC=UNIMOD:21;TA=Y;MT=Variable	10 ppm	0.8 Da

```

Comment:
Biological replicate require different source name

Reply:
I have changed it. Have a look.

Reply:
@daichengxin Can you please check now and merge it.